src/lib.rs

```diff
@@ -125,12 +124,24 @@ pub fn encode_fmt<T: Into<u128>, W: fmt::Write + ?Sized>(num: T, f: &mut W) -> f
     }
 }
 
+/// Transparent wrapper over a value, [`Display`]ing it via [`encode_fmt()`].
+///
+/// [`Display`]: fmt::Display
+#[derive(Copy, Clone, Debug)]
+#[repr(transparent)]
+pub struct Display<T>(pub T);
```

Comment:
I think I'd prefer if we make this private and define helper functions. What do you think?

```
pub fn fmt<T: Copy>(value: T) -> Fmt<T> {
    Fmt(value)
}

pub fn fmt_alt<T: Copy>(value: T) -> FmtAlternative<T> {
    FmtAlternative(value)
}
```

```suggestion
struct Display<T>(T);
```